src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
I'll start running tests on my PC with each individual opening position, and see if I can also generate a corpus of middlegame positions (to test each individually) in order to identify whether this patch is a uniform Elo gain uniformly scaling across all classes of positions - not that it needs to be, but I expect to see erratic results for some combinations of TC and position.  Such a simple process could take much CPU time to permute so I wouldn't call it easy.  Arguably any patch won't uniformly gain Elo and uniformly scale, and often such can be easily explained by a casual code review (for example, a patch to fix endgame play may scale better starting from endgame positions rather than from opening positions).

By destabilizing I mean that future changes are likely to produce erratic results in such a test.  Certainly you've demonstrated the patch scales in general.

I'm attempting to be critical of giveaway chess patches because not only is the variant weakly solved and it is [common knowledge](http://chessvariants.com/diffobjective.dir/giveaway.html) that it plays radically differently from chess (see "Pieces in Losing Chess"), there is also evidence in FICS computer rating lists that [proof-number search](http://magma.maths.usyd.edu.au/~watkins/LOSING_CHESS/losing.pdf) is an excellent approach.  To my knowledge the other variant studied anywhere close to this extent is atomic chess, which hasn't been solved nor have any new search algorithms been invented for it.  I apologize for resorting to an argument from authority and if there is extensive research, academic papers, etc. about other variants I'd be glad to read those in addition to the antichess and atomic papers I've already read.